_topic_map.yml

```diff
@@ -1706,16 +1706,18 @@ Topics:
 - Name: Disabling Windows container workloads
   File: disabling-windows-container-workloads
 ---
-Name: Sandboxed Container Support for OpenShift
+Name: Sandboxed Containers Support for OpenShift
```

Comment:
Here at the book level you have "Sandboxed Containers" capitalized but throughout the rest of the doc, it is lowercased. Does this part need to be lowercase also?

Reply:
In this instance, we can have capitalized since it's the title of the book. But the approved product title is "OpenShift sandboxed containers"